Makefile

```diff
@@ -3,18 +3,29 @@ BUILDDATE:=$(shell date -u "+%Y%m%d%H%M%S")
 HASH:=$(shell git rev-parse --short HEAD)
 FULLVER:=$(VERSION)-$(BUILDDATE)-$(HASH)
 
-.FAKE : all clean cleanswf swf js demo democlean tests dist jshint
+DEMO_DIR:=demo
+TESTS_DIR:=tests
+BUILDSCRIPTS_DIR:=buildscripts
+
+DYNAMIC_AUDIO_SWF:=assets/dynamicaudio.swf
+CORTADO_JAR:=assets/cortado.jar
+
+JS_FILES := $(shell find src/js -type f -name "*.js")
+
+.FAKE : all clean cleanswf swf js demo democlean tests dist lint
```

Comment:
Shouldn't this be `.PHONY` ? https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html